README.md

```diff
@@ -1,8 +1,36 @@
-Before you start
-=================
+# www.openshift.io
 
-Backend API
------------
+
+This repository contain the landing page content for openshift.io, a
```

Comment:
not sure I follow. you mean move the intro/community feedback/upstream development  below the build details ? 

will fix the typo ;)